Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -341,7 +341,13 @@ bool AsyncScrollingCoordinator::requestScrollToPosition(ScrollableArea& scrollab
     if (!stateNode)
         return false;
 
-    stateNode->setRequestedScrollData({ ScrollRequestType::PositionUpdate, scrollPosition, scrollType, clamping, animated });
+    if (options.animated == ScrollIsAnimated::Yes && options.scrollClass == ScrollClass::ScrollBy)
```

Comment:
Maybe we should always do the scrollBy thing for ScrollClass::ScrollBy, and not just when animated.

Reply:
Right I agree.